ruby-training/trainees/alan/test_katas/04/Arrayify/arrayify.rb

```diff
@@ -0,0 +1,12 @@
+
+
+class Arrayify
+    def self.to_a_digits_array (number)
+        array_of_digits = number.to_s.split("")
+        array_of_digits.map(&:to_i)
+    end
+
+    def self.make_it_reverse (number)
```

Comment:
Again, you're creating a method that simply wraps an existing method (just call reverse on the result somewhere else. Also, this method also does something other than "reversing" which violates the single responsibility principle as well as the method naming conventions.